hs_core/templates/pages/genericresource.html

```diff
@@ -975,6 +989,78 @@ <h4 class="modal-title">Confirm file deletion</h4>
                                 </div>
                                 {% endif %}
 
+                                  <!-- ========= CONNECTIONS TAB ========= -->
+                                {% if connections %}
+                                    <div class="tab-pane" id="connectionsTab">
+                                         {% if connections.collected_by %}
+                                                <h4><strong>Belongs to Collections:</strong></h4>
+                                                <hr style="margin-top: 0;margin-bottom: 2px">
+                                                <table id="collected-by-table" class="table-hover table-striped resource-custom-table" width="100%">
+                                                    <thead>
+                                                        <tr>
+                                                            <th>Title</th>
+                                                            <th>Owners</th>
+                                                            <th>Sharing Status</th>
+                                                            <th>My Permission</th>
+                                                        </tr>
+                                                    </thead>
+                                                    <tbody>
+                                                        {% for res in connections.collected_by %}
```

Comment:
@zhiyuli - I agree with @dtarb. If you want to open a new issue for this we could have Mauriel help. Basically, what is needed is a sentence/blurb to act as the section header for each relationship type. For example:

"isPartOf" --> "This resources belongs to the following collections:"
"isHostedBy" --> "This resource is hosted by the following system or organization:"
"isCopiedFrom" --> "This resource was copied from the following resource or source:"
"isExecutedBy" --> "This resource can be executed by the following resources:"
"isDataFor" --> "This resource is the data for the following publication:"
...